### PR TITLE
Revert "Fix filters in amp-bind reference (#27143)"

### PR DESCRIPTION
**Pull request by sebastianbenz**
The commit broke the amp.dev build due to unclosed filter.

### CODE REVIEW

File: extensions/amp-bind/amp-bind.md [hunk 1]
```diff
@@ -261,6 +261,8 @@ An `amp-state` element must contain a child `<script>` element.
 
 #### Attributes
 
+[filter formats="websites, stories, ads"]
```
Comment (caroqliu) [inline extensions/amp-bind/amp-bind.md]: It looks like this `filter` line appears both above and below the "Attributes" header with the change, the one above being unclosed.

Comment (sebastianbenz) [reply extensions/amp-bind/amp-bind.md]: You're right - (our parser doesn't care though). Went ahead and fixed the filters.

**Comment by sebastianbenz** [comment]
OK, looked through the doc and fixed a few things:

* wrong indentation for attributes and events (sidenote: I find this structure a bit confusing as amp-bind does not directly match to an amp-bind element so it might make more sense to move attributes/actions into the state section //cc @CrystalOnScript). 
* removed stories from filters as stories don't support amp-bind
* fixed duplicate filter instruction

@caroqliu PTAL

File: extensions/amp-bind/amp-bind.md [hunk 2]
```diff
@@ -257,31 +257,34 @@ An `amp-state` element must contain a child `<script>` element.
...
+## Attributes
 
-##### `src` (optional)
+### `src` (optional)
 
 The URL of the remote endpoint that must return JSON, which is used to this `amp-state`. This must be a HTTP service with a proper CORS configuration for the page. The `src` attribute allows all standard URL variable substitutions. See the [Substitutions Guide](../../spec/amp-var-substitutions.md) for more info.
 
+AMP batches XMLHttpRequests (XHRs) to JSON endpoints, that is, you can use a single JSON data request as a data source for multiple consumers (e.g., multiple `amp-state` elements) on an AMP page.
+
+For example, if your `amp-state` element makes an XHR to an endpoint, while the XHR is in flight, all subsequent XHRs to the same endpoint won't trigger and will instead return the results from the first XHR.
```
Comment (kristoferbaxter) [inline extensions/amp-bind/amp-bind.md]: Nice move, much more relevant to the section than a note toward the end.

**Comment by sebastianbenz** [comment]
Thanks for the review!

**Pull request merged**